src/mixins/transparentize.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import { isRgb, isRgba, isHsl, isHsla, isHex } from '../internalHelpers/_isValidColor'
+import hexToRgb from '../internalHelpers/_hexToRgb'
+
+const getNewTransparency = (currentTransparency: number, percentage: number) : number => (
+  currentTransparency * (1 - (percentage / 100))
+)
+
+const getColorValuesInBrackets = (color: string) : string => (
+  color.slice(color.indexOf('(') + 1, color.indexOf(')'))
+)
+
+const getValuesTillLastComma = (color: string) : string => (
+  color.slice(0, color.lastIndexOf(','))
+)
+
+const transparentizeRgb = (rgb: string, percentage: number) : string => (
+  `rgba(${getColorValuesInBrackets(rgb)}, ${getNewTransparency(1, percentage)})`
+)
+
+const getLastColorValue = (color: string) : number => (
+  Number(
+    color.slice(
+      color.lastIndexOf(',') + 1,
+      color.indexOf(')'),
+    ),
+  )
+)
+
+function transparentize(color: string, percentage : number) {
```

Comment:
Got it. I guess you don't want to use percentage then? At the moment everything is based on percentage eg. transparentize('rgba(255, 255, 255, .5)', 20) => rgba(255, 255, 255, .4)